package.json

```diff
@@ -1,21 +1,34 @@
 {
   "name": "prompts",
+  "type": "module",
   "version": "2.4.2",
   "description": "Lightweight, beautiful and user-friendly prompts",
   "license": "MIT",
   "repository": "terkelg/prompts",
   "main": "index.js",
+  "exports": {
+    ".": {
+      "require": "./index.cjs",
+      "default": "./index.js"
+    },
+    "./package.json": "./package.json"
+  },
   "author": {
     "name": "Terkel Gjervig",
     "email": "terkel@terkel.com",
     "url": "https://terkel.com"
   },
   "files": [
     "lib",
-    "index.js"
+    "dist",
+    "index.js",
+    "index.cjs"
   ],
   "scripts": {
     "start": "node lib/index.js",
+    "prebuild": "rm -rf dist",
+    "build": "tsc -p ./tsconfig.cjs.json && tsconfig-to-dual-package ./tsconfig.cjs.json",
```

Comment:
If you're interested, [pkgroll](https://github.com/privatenumber/pkgroll) offers this with zero-config.